applications/automatic-post-editing.md

```diff
@@ -2,6 +2,47 @@
 nav_order: 5
 parent: Application areas
 title: Automatic post-editing
-description: A method to correct errors in the machine translation output
-layout: coming_soon
+description: Automatic correction of machine translation errors
 ---
+
+**Automatic post-editing** (APE) is the task of correcting automatically the output of a machine translation system.
```

Comment:
*APE* should also be bold, similar to the quality estimation article.

Reply:
"automatically correcting" is a bit more idiomatic than "correcting automatically".

But maybe "editing" instead of "correcting"?

And then say "For example, automatic post-editing can be used to correct errors or apply a style."

Reply:
And maybe also contrast it to manual human post-editing?